### ISSUE TITLE
Training behavior

### ISSUE THREAD

**Issue by YicongHong** [OliverRensu/FlowAR#7]
Hi! Could you please kindly share the **training loss curve** and **FID curve** and some visualization results along the way from 0-400 training epochs of the Large or Huge model?

We wish to understand more about the training behavior of FlowAR, e.g., what overall quality to expect at different epochs, and how does the image detail refines as the training progresses.

Any information would be greatly helpful! Thanks!
